FileFormat.Heic.Decoder/FileFormat.Heic.Decoder.csproj

```diff
@@ -53,6 +53,8 @@
     <AssemblyName>FileFormat.Heic.Decoder</AssemblyName>
     <RootNamespace>FileFormat.Heic.Decoder</RootNamespace>
     <DocumentationFile>$(AssemblyName).xml</DocumentationFile>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\publish\AsposeKeyPair.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
```

Comment:
This will not work in public repo. Please make the specific configuration for that.

Reply:
I have created configurations.